_includes/integrations/react-tech-ref.md

```diff
@@ -40,6 +41,50 @@ or with Yarn
 $ yarn add @tinymce/tinymce-react
 ```
 
+### Using TinyMCE React integration in a Bootstrap dialog
+
+To use {{site.productname}} React instances inside Bootstrap UI dialogs, add the following React effect to a component that renders with the editor. This code is required because Bootstrap blocks all `focusin` calls from elements outside the dialog.
```

Comment:
It's like that because I copied from TinyMCE's docs:
> To render TinyMCE instances inside Bootstrap UI dialogs, add the following code:

I will fix it.